### PR TITLE
Add support for async endpoint in ID verification

### LINKED ISSUE #24: Support for `async_id_verification`
Hi,

Currently, there is no obvious way to use asynchronous API for ID verification (`https://api.smileidentity.com/v1/async_id_verification`).

That part of URL is hardcoded here: 

https://github.com/smileidentity/smile-identity-core-ruby/blob/master/lib/smile-identity-core/id_api.rb#L74

### PR DESCRIPTION
**Pull request by karniv00l**
Closes #24

### CODE REVIEW

File: lib/smile-identity-core/id_api.rb [hunk 1]
```diff
@@ -5,7 +5,6 @@ def initialize(partner_id, api_key, sid_server)
       @partner_id = partner_id.to_s
       @api_key = api_key
 
-      @sid_server = sid_server
```
Comment (karniv00l) [inline lib/smile-identity-core/id_api.rb:8]: not used

**Review by beaglebets — APPROVED** [review]

**Comment by beaglebets** [comment]
Thank you for contributing to our library.

**Comment by karniv00l** [comment]
Hi @beaglebets, are you guys planning to merge and release a new version maybe?

**Comment by ArlonTM** [comment]
> Hi @beaglebets, are you guys planning to merge and release a new version maybe?

@karniv00l Thanks again for your contribution. This has now been merged and a new version will soon be released.

**Pull request merged**